tests/schemas/test_openapi.py

```diff
@@ -1162,6 +1162,18 @@ def test_schema_rendering_to_json(self):
         assert b'"openapi": "' in ret
         assert b'"default": "0.0"' in ret
 
+    def test_schema_rendering_to_yaml(self):
+        patterns = [
+            path('example/', views.ExampleGenericAPIView.as_view()),
+        ]
+        generator = SchemaGenerator(patterns=patterns)
+
+        request = create_request('/')
+        schema = generator.get_schema(request=request)
+        ret = OpenAPIRenderer().render(schema)
+        assert b"openapi: " in ret
```

Comment:
can you explain the asserts please?  also it would be nice to see some additional tests if possible

Reply:
sure. this test renders openapi yaml definition from an example schema which has a duration field with a default value timedelta(0) and asserts is checking for "openapi" and the duration fields default value "0.0".

I think we can add checking the" !!python/object/apply:datetime.timedelta" tag is not in response. 
please let me know if you have any suggestions for cases it would be a great.

Reply:
I have added test case for rendering yaml with durationfield having minvalidator. Please check.